en/mkdocs.yml

```diff
@@ -61,6 +61,8 @@ nav:
     - Integrations:
           - Integration Concepts: integrations/integration-concepts.md
           - Build Your First Integration: integrations/build-your-first-integration.md
+    - Remote Apps:
+          - Connect Remote Applications: remote-apps/connect-remote-app.md
```

Comment:
Always use the same name as the topic to the file name. `connect-remote-application.md`

Reply:
Done